packages/unlock-js/src/uniswapService.ts

```diff
@@ -26,125 +26,85 @@ export class UniswapService {
       throw new Error(`No network found for the id: ${networkId}`)
     }
     const provider = new ethers.providers.JsonRpcProvider(
-      networkConfig.provider
+      'https://mainnet.infura.io/v3/957ccfce160e46c3bbc6906d2e72f2de'
```

Comment:
Please make this is a config option

Reply:
ah sorry, it was stashed between changes. I was checking if our RPC provider was slow.